axlearn/common/compiler_options.py

```diff
@@ -52,6 +52,11 @@ def default_xla_options(
             # concurrently with gradient computation for the following layer.
             xla_tpu_enable_data_parallel_all_reduce_opt="true",
             xla_tpu_data_parallel_opt_different_sized_ops="true",
+            # If MegaScale Runtime Error is encountered when running multi-slice jobs,
+            # enabling this flag will allow for termination of the job, triggering
+            # the process to exit. This is set to true to prevent the job from
+            # silently hanging and to reduce time to recovery.
+            megascale_abort_on_hangs="true",
```

Comment:
Is it a XLA flag? Curious since other xla flags have `xla_` prefix

Reply:
This is not a XLA compiler flags, but rather a libtpu runtime flag. As long as it eventually pass into LIBTPU_INIT_ARGS it should work.

Reply:
IIRC, this won't work. Could you test the AOT compilation script to confirm?
I've been using some other megascale flags which don't work here.